README.md

```diff
@@ -8,131 +8,88 @@
-* [Text Spans](docs/view/TextSpans/README.md)
-* [Toast](docs/view/Toast/README.md)
-
-## Usage
-
-### Radii
-
-The Backpack radii tokens are available as [dimension resource](Backpack/src/main/res/values/backpack.radii.xml).
-
-The supported tokens are
-
-+ `bpkBorderRadiusSm`
-+ `bpkBorderRadiusPill`
-
-### Elevation
-
-The Backpack elevation tokens are available as [dimension resource](Backpack/src/main/res/values/backpack.elevation.xml).
-
-The supported tokens are
-
-+ `bpkElevationXs`
-+ `bpkElevationSm`
-+ `bpkElevationBase`
-+ `bpkElevationLg`
-+ `bpkElevationXl`
-
-### Text Styles
-
-The Backpack text styles are available as [style resources](Backpack/src/main/res/values/backpack.text.xml).
-
-The supported styles are
-
-+ `bpkTextXs`
-+ `bpkTextXsEmphasized`
-+ `bpkTextSm`
-+ `bpkTextSmEmphasized`
-+ `bpkTextBase`
-+ `bpkTextBaseEmphasized`
-+ `bpkTextLg`
-+ `bpkTextLgEmphasized`
-+ `bpkTextXl`
-+ `bpkTextXlEmphasized`
-+ `bpkTextXl`
-+ `bpkTextXlEmphasized`
-+ `bpkTextXxl`
-+ `bpkTextXxlEmphasized`
-
-### Color
-```xml
-<TextView
-  android:text="This is Backpack Blue 500!"
-  android:textColor="@color/bpkSkyBlue" />
+```gradle
+implementation 'net.skyscanner.backpack:backpack-compose:x.x.x' // not released yet
 ```
 
+All Compose components must to be used within `BpkTheme` scope:
+
 ```kotlin
-R.color.bpkSkyBlue
+BpkTheme {
+    BpkText("Hello from Compose!")
+}
 ```
-### Gradient
 
-The Backpack gradient component is available with the `BPKGradient` utility class. It accepts the direction of the gradient as an optional parameter.
+## Features
 
-```java
-BpkGradients.getPrimary(context);
-BpkGradients.getPrimary(context, GradientDrawable.Orientation.LEFT_RIGHT);
-```
+### Android View system
+
+You can find the list of the available components, as well as the code samples and the screenshots [here](https://backpack.github.io/components/text?platform=android).
+
+All design system tokens exist as public Android resources. Here's the list of the token types and the samples of its usage:
+
+* border radii – `@dimen/bpkBorderRadiusSm` (XML), `R.dimen.bpkBorderRadiusSm` (Java/Kotlin)
+* colours – `@color/bpkSkyBlue` (XML), `R.color.bpkSkyBlue` (Java/Kotlin)
+* elevations – `@dimen/bpkElevationSm` (XML), `R.dimen.bpkElevationSm` (Java/Kotlin)
+* spacings – `@dimen/bpkSpacingLg` (XML), `R.dimen.bpkSpacingLg` (Java/Kotlin)
+* text styles – `?bpkTextBaseSize` (XML)
+
+### Jetpack Compose
+
+> Backpack for Jetpack Compose status is experimental.
+
+Backpack for Compose targets **stable Compose 1.0.0** release.
+
+You can find the list of the available components, as well as the code samples and the screenshots [here](https://backpack.github.io/components/text?platform=compose).
+
+The design system tokens are located in a `net.skyscanner.backpack.compose.tokens` package.
+Here's the list of the token types and the samples of its usage:
+
+* border radii – `BpkBorderRadius.Lg`
+* colours – `BpkColor.SkyBlue`
```

Comment:
can you also link/add the semantic colours? these should be used when possible, rather than direct access to the colours